locales/en-US/notes.properties

```diff
@@ -1,5 +1,34 @@
-welcomeTitle2=Hello!
-welcomeText2=Welcome to this one-page notepad built in to Firefox. Browse the web. Jot down notes. It’s that simple.
+welcomeTitle3=Welcome to Firefox Notes!
```

Comment:
Note that you don't have to add 3 to all the strings, only to this one since you already had a welcomeTitle in the file

Reply:
I know I did it so that we know they are linked together and that it is this version of the boilerplate copy.

Reply:
OK, but what happens if you need to change a string in the future?

Reply:
You are right I made the change.